### DISCUSSION TITLE
[New Component] [Elstatistic] 新增统计数据组件 Added the statistics component

### DISCUSSION THREAD

**Discussion by webvs2** [element-plus/element-plus#10212] [Feature Request]
Feature type: **`New Component`**

## Details

- New component name: `Elstatistic`

### Features / 特性
1. You can count down. / 可以倒计时
2. Can be used to highlight a number or a group of numbers, such as display value, such as amount, ranking, etc. / 用于突出某个或某组数字时，如显示数值，如金额，排名等。

### Living examples (if any) / 已有的例子 (如果有的话)
https://www.antdv.com/components/statistic 

### Additional comments

希望可以参与其中贡献，为前端提供绵薄之力
I hope I can participate in the contribution and provide a small contribution to the front end

**Comment by webvs2** [comment]
New components have been developed

**Reply by webvs2** [reply]
@chenxch  @sxzz  what's your opinion?

**Reply by chenxch** [reply]
Agree, but may need to wait until a major version update.

**Reply by webvs2** [reply]
Wow, thank you

**Reply by webvs2** [reply]
Should I do some preliminary work now, such as putting code into the development branch or writing api documentation

**Reply by chenxch** [reply]
Don't worry, we also need to see what other members have to say.

**Reply by webvs2** [reply]
OK, I'll keep an eye on that

**Comment by btea** [comment]
cc @element-plus/backers

**Comment by chenxch** [comment]
@jw-foss  WDYT?

**Comment by jw-foss** [comment]
The idea is good, and I personally appreciate that you are willing to make an effort on this, I think you can get started on doing it if you would like to do so, but I think it would be better for me to communicate with our designer so that we can keep the UI consistent. 

I'll come back later to see if we have the resources on the design side. Thanks!

**Reply by webvs2** [reply]
Thank you for agreeing, my friend

**Comment by huangzhongzhong** [comment]
后续我将提供Elstatistic 的UI设计稿给你，请稍等几天~

**Reply by webvs2** [reply]
谢谢，我将保持关注😄

**Comment by ryuhangyeong** [comment]
Good!

**Comment by webvs2** [comment]
@chenxch Is there a command that generates the sample document directory?

**Reply by chenxch** [reply]
```shell
pnpm run gen
```
Since it is very old and no new components have been added, this script is still `optionsAPI`, if you use it now, you need to manually modify it to `script setup`, and the test file should be changed to `tsx`.

If you're not in a hurry, you can wait for me to update it.

**Reply by webvs2** [reply]
Thanks, there's no rush, and there seems to be a lack of demo file creation when it's added.  Maybe I wasn't paying attention.  And do I observe that third party polyfill should be placed at utils?Like lodash

**Reply by chenxch** [reply]
It depends on what kind of file it is, if `lodash` we already have `lodash-unified` installed.

**Reply by chenxch** [reply]
gen updated, via #10641
